src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Post } from './types/post';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
```

Comment:
[CODE STYLE] - This function is a helper function that is used more than once in this file. Consider moving it to a separate file and import it here. This will improve the readability and maintainability of your code.